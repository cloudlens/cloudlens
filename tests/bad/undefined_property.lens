var state = "green";

match (
    "My color is (?<color>[^ ]+)!"
)


process (e) {
    if ((e.color=="blue" && state != "green" && state != undefined) ||
        (e.color=="red" && state != "blue") ||
        (e.color=="green" && state != "red")) {
          print("error:" + state + "," + e.color)
    };
    state=e.color;
    print(bogus_set[state]);
}
