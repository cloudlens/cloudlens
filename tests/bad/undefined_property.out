Error: tests/bad/undefined_property.lens, block starting line 8:
<eval>:8 TypeError: Cannot read property "blue" from undefined
